core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -181,6 +182,7 @@ class RDDStorageInfo private[spark](
     val partitions: Option[Seq[RDDPartitionInfo]])
 
 class RDDDataDistribution private[spark](
+    val executorId: String,
```

Comment:
Are we changing model in api/v1? Seems like the change is needed, but we may also need to check where it doesn't break existing one.

Reply:
the api versioning requirements are listed here: https://spark.apache.org/docs/latest/monitoring.html#api-versioning-policy

> New fields may be added to existing endpoints

so its OK to add new fields.

but, if we're adding things that really don't make sense as part of the api, then maybe we should just store a different object instead.

Reply:
Hi @squito, thanks for providing the api versioning requirements. That's really helpful.